services/opentsdb/service.go

```diff
@@ -225,7 +257,7 @@ func (s *Service) Statistics(tags map[string]string) []models.Statistic {
 }
 
 // Err returns a channel for fatal errors that occur on the listener.
-func (s *Service) Err() <-chan error { return s.err }
+// func (s *Service) Err() <-chan error { return s.err }
```

Comment:
Eurgh sorry I meant to remove this function. Nothing calls it. I don't think we need it on the services do we @jwilder?